src/servers/src/http/prometheus.rs

```diff
@@ -48,10 +51,13 @@ pub async fn remote_write(
 ) -> Result<(StatusCode, ())> {
     let request = decode_remote_write_request(body).await?;
 
-    handler
-        .write(params.db.as_deref().unwrap_or(DEFAULT_SCHEMA_NAME), request)
-        .await?;
+    let ctx = if let Some(db) = params.db {
+        Arc::new(QueryContext::with(DEFAULT_CATALOG_NAME, &db))
```

Comment:
prom always use the default catalog?

Reply:
yes, catalog is not set as of right now, it's somehow related to cloud or tenant